code/modules/economy/shippingmarket.dm

```diff
@@ -8,7 +8,7 @@
 	var/time_until_shift = 0.0
 	var/demand_multiplier = 2
 	var/list/active_traders = list()
-	var/max_buy_items_at_once = 20
+	var/max_buy_items_at_once = 99
```

Comment:
this will have some considerable balance implications.

Reply:
Yeah, I was going back and forth myself. 99 is a traditional value for the "max" to buy in games, plus it fits well with the 99 + invoice for max crate capacity. But, it could have balance issues like you said, but that's why I nerfed some of the things using unlimited purchase mats like fabric, etc.